examples/CExample/README.md

```diff
@@ -0,0 +1,51 @@
+## Example Description
+
+This C example demonstrates basic usage of the Web5 SDK functions through the C bindings. Specifically, it showcases:
+
+1. Resolving a `did:dht` identifier
+2. Printing the resolved DID document or an error message
+
+This example demonstrates how to resolve a DHT-based DID using the Web5 SDK in C. It uses the `did_dht_resolve` function with a hardcoded DID URI and the default gateway. The program then prints either the resolved DID document or an error message to the console.
+
+
+## Running the Example
+
+To run this C example:
+
+1. Ensure you have a C compiler installed (e.g. GCC) and [Just](https://github.com/casey/just) command runner.
```

Comment:
fwiw we already have `just` included [in the hermit dependencies](https://github.com/TBD54566975/web5-rs/blob/main/bin/.just-1.23.0.pkg), though I'd be curious if hermit has a `gcc` dependency available b/c if not then the developer will have to bring-their own which would be the first of it's kind in this repo

Reply:
> fwiw we already have just included in the hermit dependencies,

yay thanks didn't know that. I'll update accordingly. 

> though I'd be curious if hermit has a gcc dependency available b/c if not then the developer will have to bring-their own which would be the first of it's kind in this repo

when I checked, I looked only at https://cashapp.github.io/hermit/sdks/ and didn't see any C SDKs. Upon closer inspection, I see that there is `macos-cross-gcc-aarch64-unknown-linux-musl` is available. I'm concerned that it is macOS only and is meant for cross-compilation. What are your thoughts on this?

Reply:
Yeah I think sticking with byo-gcc is fine, so long as it's called out